src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -250,7 +260,8 @@ export const StakeInput: React.FC<StakeInputProps & StakeRouteProps> = ({
       stakingAssetAccountId,
       stakingAssetId,
       stakingAmountCryptoBaseUnit: toBaseUnit(amountCryptoPrecision, stakingAsset.precision),
-      runeAddress,
+      // typescript is borked, one of them is defined because of the early return
+      runeAddress: currentRuneAddress || runeAddress || '',
```

Comment:
Add me to `useRfoxStake()` too or MM re-deposits will be borked

Reply:
Done in [f79b7b4](https://github.com/shapeshift/web/pull/7734/commits/f79b7b42a0882d52ddcd290fad0162f018b24ff9)

Seems like you where right! It was the pita